### PR TITLE
Create Massengeschmack.tv

### PR DESCRIPTION
**Pull request by Mertsch**

### CODE REVIEW

File: src/chrome/content/rules/Massengeschmack.tv.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<ruleset name="massengeschmack.tv">
+	<target host="massengeschmack.tv" />
```
Comment (J0WI) [inline src/chrome/content/rules/Massengeschmack.tv.xml]: alpha.massengeschmack.tv
beta.massengeschmack.tv
cache.massengeschmack.tv
dl.massengeschmack.tv
forum.massengeschmack.tv (mixed content issues)
www.massengeschmack.tv

**Comment by J0WI** [comment]
Wildcards and exclusions requires test urls. We generally recommend to just list all targets.

**Comment by Mertsch** [comment]
@J0WI Updated the ruleset

**Comment by J0WI** [comment]
I still miss `alpha.massengeschmack.tv`, `beta.massengeschmack.tv` and a comment for the mixed content issues on `forum.massengeschmack.tv`

**Comment by J0WI** [comment]
@Mertsch please have a look at https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#non-working-hosts

File: src/chrome/content/rules/Massengeschmack.tv.xml [hunk 2]
```diff
@@ -11,8 +11,13 @@
     <target host="alpha.massengeschmack.tv" />
     <target host="beta.massengeschmack.tv" />
 
-	<target host="forum.massengeschmack.tv" platform="mixedcontent"/>
-
+<!--
+    Mixed content:
+        - Images from schnittbrot.com *
+        - Images from www.schnittbrot.com *
```
Comment (J0WI) [inline src/chrome/content/rules/Massengeschmack.tv.xml]: The host contains also mixed script, so We can't keep this target active without breaking things.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**